arEd/blogs/templates/add_blog.html

```diff
@@ -0,0 +1,8 @@
+<h1>Add your blog here !</h1>
```

Comment:
the templates folder should not be inside the blogs folder, it should be inside the root folder